### PR TITLE
Doc : Adds instructions on adding and removing environments and metadata

### PR DESCRIPTION
**Pull request by UjjwalAyyangar**
Addresses #266

### CODE REVIEW

**Comment by UjjwalAyyangar** [comment]
The description for the issue has changed. I'll make the changes asap.

File: docs/protocol/getting_started.md [hunk 1]
```diff
@@ -69,9 +69,9 @@ Run popper run:
 ```bash
 popper run
 ```
-To run a pipeline named exp1 :-
+To run a pipeline named myexp :-
```
Comment (ivotron) [inline docs/protocol/getting_started.md]: this one too 😄

Comment (UjjwalAyyangar) [reply docs/protocol/getting_started.md]: Oops :see_no_evil:

**Comment by ivotron** [comment]
thanks!

**Pull request merged**